package.json

```diff
@@ -61,6 +61,7 @@
     "graphql-tag": "2.10.1",
     "https-proxy-agent": ">=3.0.0",
     "jdenticon": "2.2.0",
+    "lodash": "4.17.15",
```

Comment:
we import the lodash packages individually as you can see in line 65+

Reply:
oh, sure. I even used it once..